website/docs/advanced/maintaining-uptime.md

```diff
@@ -0,0 +1,74 @@
+---
+id: maintaining-uptime
+title: Maintain validator uptime with systemd or Docker
+sidebar_label: Maintain Validator Uptime
+---
+
+Validators are expected to maintain connectivity with the Ethereum network 24/7. If you're running your validator on a cloud server, or if you want your validator to automatically start running when the host machine restarts, consider running your client software as a background service. Docker and systemd let you run Prysm as a background service. 
```

Comment:
> Validators are expected to maintain connectivity with the Ethereum network 24/7

This isn't necessarily true. Every validator has an assignment as an attester once every 6 or so minutes. You only need to be available when you have to attest or propose. As this is unpredictable, it is fine to say that a validator should be up as much as possible, but it is inaccurate to tell users they need 100% uptime